src/Hyperwallet/Model/User.php

```diff
@@ -46,7 +46,11 @@
  * @property string $language The user language
  * @property string $programToken The users program token
  * @property string $verificationStatus The status of user verification
+ * @property string $businessStakeholderVerificationStatus The status of Business Stakeholder verification
+ * @property string $letterOfAuthorizationStatus The status of Letter of Authorization verification
  * @property array $documents The array of documents returned for document upload
+ * @property string $governmentIdType The status of Letter of Authorization verification
+ * @property array $links The array of HATEOS links
  *
```

Comment:
The field 'timeZone' also needs to be added

Reply:
Done